index.html

```diff
@@ -160,36 +132,46 @@
     </p>
 
     <!--
-    <h2>Town Name Randomizer</h2>
-    <p>
-      <button id="generate_town">Generate</button>
-    </p>
-    <p id="random_town">
-      <span class="prefix"></span><span class="suffix"></span>
-    </p>
-    -->
-
-    <footer>
+        <h2>Town Name Randomizer</h2>
+        <p>
+          <button id="generate_town">Generate</button>
+        </p>
+        <p id="random_town">
+          <span class="prefix"></span><span class="suffix"></span>
+        </p>
+        -->
+
+    <footer class="text-center">
```

Comment:
Do you think the font size for the footer could be a bit smaller to differentiate it from the main content?